resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,95 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('has_disabled_link_warning') === 'true';
         }
     }"
-    class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
-    @click="openModal"
 >
-    <span>{{ $text ?? $slot ?? '' }}</span>
+    <a
+        :href="hasDisabledLinkWarning() ? '{{ $url }}' : 'javascript:;'"
+        :target="hasDisabledLinkWarning() ? '_blank' : '_self'"
+        class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
+        @click="hasDisabledLinkWarning() ? redirect() : openModal()"
```

Comment:
@Highjhacker It would be safe adding `rel="noopener nofollow"` when `hasDisabledLinkWarning() === true`.
Like we do here https://github.com/ArkEcosystem/laravel-ui/pull/504/files#diff-52d2df70e98c36e959bb034aa21460cecacbd5f1cb2aaa96b2ade978198818f5R90.
You can add something like this `:rel="hasDisabledLinkWarning() ? 'noopener nofollow' : ''"`

Reply:
Couldn't access `hasDisabledLinkWarning` method from there but adjusted to follow your suggestion !